### PR TITLE
feat: update schedule

### PR DESCRIPTION
**Pull request by tanhauhau**

### CODE REVIEW

**Review by serverwentdown — APPROVED** [review]

**Comment by cheeaun** [comment]
I think should really optimize/resize the images? One of the speaker photos = 3.3MB 😣

**Comment by tanhauhau** [comment]
it is big in the source code, but i think the gatsby image plugin will optimise it during build.
so if you look at the preview, the user loads a much much smaller image

**Comment by cheeaun** [comment]
@tanhauhau oh nice! I can see them in `webp` on my Chrome 🤩

Screenshot 2022-10-05 at 1 33 48 PM

👏 👏

**Comment by tanhauhau** [comment]
works of some master. i just upload picture and update data ORZ

File: src/sections/5.speakers.js [hunk 1]
```diff
@@ -291,6 +291,14 @@ const speakers = [
     bio: "Just your average script kiddie",
     twitter: "cherboon",
   },
+  {
+    name: "Ten Zhi-Yang",
+    target: "ten-zhi-yang",
+    imgId: "tenzhiyang",
+    position: "Software Developer, Shopify",
+    bio: 'I strongly believe in the saying "We build the things we can buy, so that we have the skills to build the things we can\'t buy" - Adam SavageA software developer transitioning from front end to full stack, I enjoy building small weekend projects. Code, Hardware, Carpentry, anything that tickles my fancy I will attempt. Many failures and a few successes, but most times, I will get something to talk about.',
```
Comment (cheeaun) [inline src/sections/5.speakers.js]: `Adam SavageA` seems like a typo?

**Pull request merged**